lib/data_migrate.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
 
-require File.join(File.dirname(__FILE__), "data_migrate", "data_migrator")
+require "pry"
```

Comment:
Is this `require` needed?

Reply:
oh yeah, good point! i'm still working on this branch, so need pry to debug, i'll remove them before deploying to rubygems. this branch is not production ready yet!